File: Projekt1_cpp/Projekt1_cpp/PustePole.cpp
#include "PustePole.h"

PustePole::PustePole()
	:Organizm(-1, -1, -1, false, make_pair(-1, -1), nullptr, false) {}

Organizm* PustePole::potomek(pair<int, int>pole) {
	return nullptr;
}

void PustePole::rysowanie() {
	cout << "_";
}

string PustePole::getNazwa() {
	return "puste";
}

File: Projekt2_java/src/projekt2/PustePole.java
package projekt2;

import java.awt.*;

public class PustePole extends Organizm {
    public PustePole() {
        super(-1, -1, -1, false, new Pair(-1, -1), null, false);
    }

    public Organizm potomek(Pair pole) {
        return null;
    }

    @Override
    public void akcja() {
    }

    @Override
    public void kolizja(Organizm organizm) {
    }

    @Override
    public Color rysuj() {
        return Color.WHITE;
    }

    @Override
    public String getNazwa() {
        return "puste";
    }
}
